schemas/dbt_project.json

```diff
@@ -254,6 +254,9 @@
                     "$ref": "#/$defs/boolean_or_jinja_string",
                     "default": false
                 },
+                "+contract": {
+                    "$ref": "#/$defs/boolean_or_jinja_string"
+                },
```

Comment:
`contract` is an object now, containing a single bool `enforced` (https://github.com/dbt-labs/dbt-core/issues/7184, https://github.com/dbt-labs/dbt-core/pull/7222)

Reply:
yep, this should be deleted. Per the tests in that [second PR](https://github.com/dbt-labs/dbt-core/pull/7222/files#diff-acc8cb2ee65ad9cfdd7f4555ac47b95eb4443413ac3a81c8dc913aae5eb7bc34) the contract object is nested in `config` and i added it to the `model_config` block down below